ckanext/validation/common.py

```diff
@@ -78,7 +78,7 @@ def run_validation(assume_yes, resource_id, dataset_id, search_params):
 
     if resource_id:
         for resource_id in resource_id:
-            resource = get_action('resource_show')({}, {'id': resource_id})
+            resource = get_action('resource_show')({'user': context['user']}, {'id': resource_id})
```

Comment:
`context` doesn't exist. This appears to be the code for running from the command line, so there won't be a `user`.